My_work_2/Funny matrix.py

```diff
@@ -0,0 +1,10 @@
+#%%
+import numpy as np
+
+#%%
+
+n = 10
+Arr = np.diag(np.full(n, 2))
+c = np.diag(np.full(n-1, -1), 1)
+Arr += c + c.T
```

Comment:
does c.T returns copy or a view object here? How do we check it?

Reply:
It returns a view object. I checked it with this code.
`c = np.zeros((2, 2))`
`b = c.T`
`c[0, 0] = 1000`
`print(c)`
`print(b)`

out:
`1000     0`
`      0     0`
`1000     0`
`      0     0`
It turned out that b[0,0] = 1000 too.

Reply:
That is correct XD